packages/yoastseo/src/languageProcessing/helpers/html/matchParagraphs.js

```diff
@@ -20,9 +20,11 @@ const getParagraphsInTags = function( text ) {
 	}
 
 	// Returns only the text from within the paragraph tags.
-	return map( paragraphs, function( paragraph ) {
+	paragraphs = map( paragraphs, function( paragraph ) {
 		return paragraph[ 1 ];
 	} );
+
+	return paragraphs.filter( paragraph => paragraph.length > 0 );
```

Comment:
Added a new step to filter out empty paragraphs. This is to prevent `[ "" ]` as a valid output.